CONTRIBUTING.md

```diff
@@ -20,10 +20,12 @@ You can follow these steps:
  1. Make sure you have Poetry installed ([see instructions here](https://python-poetry.org))
  2. Clone the Textual repository
  3. Run `poetry shell` to create a virtual environment for the dependencies
- 4. Run `poetry install` to install all dependencies
+ 4. Run `make setup` to install all dependencies
  5. Make sure the latest version of Textual was installed by running the command `textual --version`
  6. Install the pre-commit hooks with the command `pre-commit install`
```

Comment:
An alternative would be to `make install-pre-commit`, and that `make` target handles ensuring the venv is used for that.